src/mono/CMakeLists.txt

```diff
@@ -575,6 +574,19 @@ endif()
 ### End of debug build checks
 
 
+######################################
+# OS SPECIFIC CHECKS
+######################################
+if(ENABLE_NETCORE)
+  if(TARGET_IOS OR TARGET_ANDROID)
```

Comment:
Does it imply tvOS as well?

Reply:
yes, see https://github.com/dotnet/runtime/blob/d8b2120df4759b050a50d88431777ae0870caac2/src/mono/CMakeLists.txt#L283-L285